### PR TITLE
Enable Reps API in contentSDk.

### PR DESCRIPTION
**Pull request by scottliubox**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @scottliubox has signed the CLA. Thanks for the pull request!

File: BoxContentSDK/BoxContentSDKTests/BOXRequestTestCase.m [hunk 1]
```diff
@@ -172,4 +172,10 @@ - (NSArray *)itemsFromResponseData:(NSData *)data
     return [expectedItems copy];
 }
 
+- (NSDictionary *)cannedRepresentationHintHeader
+{
+    return [NSDictionary dictionaryWithObject:@"[original][pdf][hls][mp4][mp3][png,jpg]"
```
Comment (jlawton) [inline BoxContentSDK/BoxContentSDKTests/BOXRequestTestCase.m]: Dictionary literal syntax might be easier to read here.

**Review by jlawton — APPROVED** [review]

**Comment by jlawton** [comment]
The X-Rep-Hints header may well change a bit as we check all cases, and it'd be a good idea to get it reviewed by acharles.

**Comment by Prithvijutur** [comment]
This is no longer applicable

**Pull request closed without merging**